### ISSUE TITLE
The messages window does not always scroll to bottom on ordinary messages

### ISSUE THREAD

**Issue by robertwesterlund** [JabbR/JabbR#50]
This problem at least occurs on IE9.

Haven't found anything in common between the times it happens, but once I find repro steps I will update this issue.

**Comment by kathryn-thompson** [comment]
I have this problem in Chrome 21.0.1180.89  on OSX

**Comment by dpatti** [comment]
I've seen this a few times over the past few days on Chrome 22. On further inspection, I have the following values:

```
this[0].scrollTop = 2085
this.height() = 523
this[0].scrollHeight = 2609
```

Which means you get `2085 + 523 >= 2609`, or `2608 >= 2609`, and so `isNearTheEnd()` fails, and the messages don't scroll. If I scroll down manually, it will not fix it; future messages will still not cause scrolling. However, if I scroll down and press Enter (trying to send an empty message), I notice that it bumps it down 1 pixel, and then I receive messages normally.

To recap: using the scrollbar or scrollwheel to get to the bottom, for some reason, puts you 1 pixel off of what JabbR considers "the bottom". Chatting or pressing Enter forces the scroll to the absolute bottom. Obviously you can just alter how `isNearTheEnd()` works to account for the 1px discrepancy, but I'm wondering if there is a better solution.

**Comment by davidfowl** [comment]
This is fixed now.
